plugins/sidecar-injector/internal/lifecycle/lifecycle.go

```diff
@@ -137,16 +137,16 @@ func (impl Implementation) reconcileMetadata(
 		},
 	}
 
-	// Add USERNAME and PASSWORD environment variables from secret defined in configuration
-	credentialSecretName := configuration.DocumentDbCredentialSecret
-	log.Printf("Adding USERNAME and PASSWORD environment variables from secret '%s'", credentialSecretName)
+	// Add USERNAME and PASSWORD environment variables from secret
+	// TODO: Make this configurable and expose it in the configuration
+	log.Printf("Adding USERNAME and PASSWORD environment variables from secret")
 	envVars = append(envVars,
 		corev1.EnvVar{
 			Name: "USERNAME",
 			ValueFrom: &corev1.EnvVarSource{
 				SecretKeyRef: &corev1.SecretKeySelector{
 					LocalObjectReference: corev1.LocalObjectReference{
-						Name: credentialSecretName,
+						Name: "documentdb-credentials",
```

Comment:
Why are we hardcoding it again? If not provided, it automatically looks for the default. Please revert.